### PR TITLE
Convert to Actions

### PR DESCRIPTION
**Pull request by jason-jackson**
### Changed (non-breaking)
- Convert to Github actions

---

### Feature PR Checklist
- [x] Documentation (README, local.env.dist, etc.)
- [ ] Unit tests created or updated (see `test.bats` file)

### Release PR Checklist
- [ ] Update version number in `ecs-deploy` script
- [ ] Give the PR a title of `Release _._._ - Summary of change` (using whatever
      the new version number is, plus a succinct summary of what changed)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,19 +172,16 @@ this script.
...
-Using the ```-e``` argument you can provide the name of an environment variable that
-holds the value you wish to use for the tag. On Codeship they set an env var named CI_TIMESTAMP.
+Using the `-e` argument you can provide the name of an environment variable that
+holds the value you wish to use for the tag.
 
-So we use ```ecs-deploy``` like this:
+For example:
 
-    ecs-deploy -c production1 -n doorman-api -i my.private.repo/doorman-api -e CI_TIMESTAMP
+    ecs-deploy -c CLUSTERNAME -n SERVICENAME -i my.private.repo.com/frontend_container:latest -e CI_TIMESTAMP
```
Comment (briskt) [inline README.md:184]: If you're using the `-e` option, you probably shouldn't supply a tag (e.g. `latest`).

File: README.md [hunk 2]
```diff
@@ -172,19 +172,16 @@ this script.
 Use Environment Variable for tag name value
 -------------------------------------------
 In some cases you may want to use an environment variable for the tag name of your image.
-For instance, we use Codeship for continuous integration and deployment. In their Docker
-environment they can build images and tag them with different variables, such as
-the current unix timestamp. We want to use these unique and changing values for image tags
-so that each task definition refers to a unique docker image/tag. This gives us the
-ability to revert/rollback changes by just selecting a previous task definition and
-updating the service. We plan to add a revert command/option to ecs-deploy to simplify this further.
+For example, we want to use a unique docker image/tag for each task definition. This
```
Comment (briskt) [inline README.md:175]: looks good

**Review by briskt — CHANGES_REQUESTED** [review]
needs one tweak to the README

**Review by briskt — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -172,19 +172,16 @@ this script.
...
-Using the ```-e``` argument you can provide the name of an environment variable that
-holds the value you wish to use for the tag. On Codeship they set an env var named CI_TIMESTAMP.
+Using the `-e` argument you can provide the name of an environment variable that
+holds the value you wish to use for the tag.
 
-So we use ```ecs-deploy``` like this:
+For example:
 
-    ecs-deploy -c production1 -n doorman-api -i my.private.repo/doorman-api -e CI_TIMESTAMP
+    ecs-deploy -c CLUSTERNAME -n SERVICENAME -i my.private.repo.com/frontend_container -e CI_TIMESTAMP
```
Comment (forevermatt) [inline README.md:184]: Using all caps for the example cluster name and service name makes it seem like they might also be environment variable names (like the `-e` tag). What would you think of something like this?
```suggestion
    ecs-deploy -c my-cluster-name -n my-service-name -i my.private.repo.com/frontend_container -e CI_TIMESTAMP
```

Comment (jason-jackson) [reply README.md:184]: Sounds good, I copied those from a few lines above it, I'll make another PR for README changes only.

**Pull request merged**